src/db.c

```diff
@@ -516,27 +506,29 @@ static inline unsigned long dictSizebySlot(redisDb *db, int slot, dbKeyType keyT
  * In this case slot #3 contains key that we are trying to find.
  * 
  * This function is 1 based and the range of the target is [1..dbSize], dbSize inclusive.
- * Time complexity of this function is O(log^2(CLUSTER_SLOTS))
- * */
+ * 
+ * To find the slot, we start with the root node of the binary index tree and search through its children from the highest index (2^14 in our case) to the lowest index. 
+ * At each node, we check if the target value is greater than the node's value. 
+ * If it is, we remove the node's value from the target and recursively search for the new target using the current node as the parent.
```

Comment:
```suggestion
 * To find the slot, we start with the root node of the binary index tree and search through its children
 * from the highest index (2^14 in our case) to the lowest index. At each node, we check if the target 
 * value is greater than the node's value. If it is, we remove the node's value from the target and recursively
 * search for the new target using the current node as the parent.
```
Just fix some spacing.